DEVELOPMENT.md

```diff
@@ -155,4 +155,9 @@ Every time the version number is incremented, these parts of the codebase need t
 
 ## Documentation Style Guide
 
-cleanlab-studio uses [Google style](https://google.github.io/styleguide/pyguide.html#381-docstrings) docstrings ([example](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html))
+We autogenerate documentation for the `studio` module from docstrings. In order for the generated docs to be formatted correctly, use [Google style](https://google.github.io/styleguide/pyguide.html#381-docstrings) docstrings ([example](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html))
+
+### Relative Link Formatting
+
+- To link to other functions in the `studio` module within a docstring, use the following format: `[link text](#method-function_name)`
+- To link to another page in Cleanlab Studio docs use `[link text](../../{path within cleanlab-studio-docs/docs})`
```

Comment:
Should these be relative links, or would absolute links like`/guide` also work and be more convenient?

Reply:
Ah yeah I think absolute paths from within the docs folder should work